<launch>
    <include file="$(find esvr2_ros)/launch/other/simulation.launch">
        <arg name="required" value="true"/>
    </include>
    <include file="$(find esvr2_ros)/launch/other/esvr2.launch">
        <arg name="ros_input_type" value="STEREO_SPLIT"/>
        <arg name="enable_pivot_controller" value="true"/>
        <arg name="required" value="true"/>
    </include>
</launch>